src/cljdoc/render/api.clj

```diff
@@ -51,24 +53,24 @@
     (str (ns-link-fn (if ns (util/replant-ns current-ns ns) current-ns))
          (when var (str "#" var)))))
 
-(defn render-doc [mp render-wiki-link]
+(defn render-doc [mp render-wiki-link fix-opts]
```

Comment:
Given that we now pass down `render-wiki-link` and `fix-opts` all the way down to render the `doc-str` maybe it would make sense to instead pass down a function that takes `doc-str` and returns html? Might require some renaming of `docstring->html` but seems like it might be worth it? 

Ideally the "constructor" for that function would be easy to use within REPL sessions.

Reply:
I'd also be perfectly ok with merging this first and creating an issue to follow up with a small refactor.

Reply:
I like that idea, I'll take a look.

Reply:
You know what? Let's try to get this to master while we have momentum. 🙂  (A follow up issue makes sense).

Reply:
Sounds good to me! 🙂 🚀